v-next/hardhat-utils/test/fixture-projects/sub-process/sub-process.ts

```diff
@@ -0,0 +1,23 @@
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+
+import { writeJsonFile } from "../../../src/fs.js";
+
+export const ABSOLUTE_PATH_TO_TMP_SUBPROCESS_FILE: string = path.join(
+  path.dirname(fileURLToPath(import.meta.url)),
+  "tmp-file.json",
+);
+
+const subprocessInfo = {
+  executed: false,
+  arg1: "",
+  arg2: "",
+};
+
+(async () => {
```

Comment:
Why async here?

Reply:
I guess you wanted to introduce a small delay, but I believe this gets immediately executed, because there's nothing else in the event loop

Reply:
Actually it's just a distraction error, it was not supposed to be there. Removed [here](https://github.com/NomicFoundation/hardhat/pull/5471/commits/a1354fca15ac2aa95bc5082f157cc5338bcc29c5)